rust/routee-compass-powertrain/src/routee/vehicle/default/ice.rs

```diff
@@ -36,6 +35,9 @@ impl VehicleType for ICE {
     fn name(&self) -> String {
         self.name.clone()
     }
+    fn state_variable_names(&self) -> Vec<String> {
+        vec![String::from("energy")]
```

Comment:
Similar to the bev, this could probably become `energy_liquid` and we could pull out `energy` from the config.